tap_shopify/streams/products.py

```diff
@@ -17,7 +17,7 @@ class Products(Stream):
 
     products_gql_query = """
         query GetProducts($query: String, $cursor: String) {
-            products(first: 50, after: $cursor, query: $query) {
+            products(first: 20, after: $cursor, query: $query) {
```

Comment:
I had to bump the number of products down here as the query is getting large and shopify returns a cost error:

```
Exception: [{'message': 'Query cost is 1192, which exceeds the single query max cost limit (1000).\n\nSee https://shopify.dev/concepts/about-apis/rate-limits for more information on how the\ncost of a query is calculated.\n\nTo query larger amounts of data with fewer limits, bulk operations should be used instead.\nSee https://shopify.dev/api/usage/bulk-operations/queries for usage details.\n', 'extensions': {'code': 'MAX_COST_EXCEEDED', 'cost': 1192, 'maxCost': 1000, 'documentation': 'https://shopify.dev/api/usage/rate-limits'}}]
```